### DISCUSSION TITLE
HACK recent Y cam

### DISCUSSION THREAD

**Discussion by yodo86** [roleoroleo/yi-hack-Allwinner-v2#538] [Q&A]
HI 
Is it possible tio use this hack or another, with the cam in pictures ?

I cannot find any information about yhi model but it work fine with YI Iot APP and thelast  native firmware is 6.0.05.10.202208191758
![model](https://user-images.githubusercontent.com/35577638/205712513-a2b57cca-6ca3-4405-8084-0b09da63e81b.jpg)
![face](https://user-images.githubusercontent.com/35577638/205712552-49008efe-2b02-4964-a241-d3fdb562f66c.jpg)

thanks

**Comment by roleoroleo** [comment]
Try to make a backup. If it works, I will check the model.

**Comment by yodo86** [comment]
Hello
this link
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)
doesnt work with mine
Anything else but using soldering ?

**Reply by roleoroleo** [reply]
If the link doesn't work, the cam needs a different hack.
